### DISCUSSION TITLE
Consulta de los puntos 1 y 2 de QueMePongo VI

### DISCUSSION THREAD

**Discussion by atta25** [dds-jv/foro#138] [Consultas de objetos] [answered]
1) _Como usuarie de QueMePongo quiero tener una sugerencia diaria de qué ponerme y que  todas las mañanas, diariamente, esta sea actualizada_
2) _Como empleade de QueMePongo quiero poder disparar el cálculo de sugerencias diarias para todos los usuarios para poder ejecutar esta acción a principio de cada día._

Tengo 2 consultas, la primera es para saber si lo estoy interpretando bien, para que un usuario tenga su sugerencia diaria primero el empleado tuvo que disparar el cálculo de sugerencias, no?
Y la segunda (si fuese como lo pienso en la primera consulta), el punto 1) habla de **una sugerencia diaria** pero el cálculo de sugerencias implica obtener una colección de sugerencias, cuál de todas debería sugerirse al usuario?

**Comment by roli93** [comment, marked as answer]
Buenas @atta25 !

1) Sí, una vez por dia el empleado va a disparar ese proceso y ahí se calcularán las sugerencias diarias de cada usuario
2) Se refiere a toda la lista de atuendos sugeridos. Modelar cada atuendo sugerido como `Sugerencia` o llamar `Sugerencia` a la lista de todos los atuendos sugeridos es algo que podés hacerlo como te quede más cómodo a nivel código, pero el requerimiento se refiere a toda la lista.

**Comment by atta25** [comment]
Perfecto! Gracias.
